### ISSUE TITLE
2.3.2 not working on Nightly

### ISSUE THREAD

**Issue by Smultie** [dirkjanm/firefox-send-to-xbmc#50]
When using the latest version of the addon on the latest version of Firefox (Nightly) I get the following error when sending to Kodi 
'You have to set up your Kodi address first in the Addon Settings'.

At first I thought the settings were just erased during the update of the addon, but when I tried to enter the preferences through about:addons , I got the following error:

'File not found

Firefox can’t find the file at resource://jid0-ycm0p5wlcgjvbjczhausq5h26wm-at-jetpack/send-to-xbmc/data/preferences.html.

    Check the file name for capitalization or other typing errors.
    Check to see if the file was moved, renamed or deleted.'

**Comment by dirkjanm** [comment]
I've found the bug and am working on a fix. Could you check if you can access the preferences via right click on a supported url -> send to kodi -> manage servers? Are the settings reset there since the latest update?

**Comment by Smultie** [comment]
Hi @dirkjanm , rightclicking works!

**Comment by dirkjanm** [comment]
Cool, I've fixed this in 2.3.3, which should be available once it is approved on AMO (probably today or tomorrow).

**Comment by Nalhagop** [comment]
OK.  I opened another ticket for this issue.  I'll close it now.  Thank you!

**Comment by Nalhagop** [comment]
2.3.3 works perfectly.  Thank you!

**Comment by dirkjanm** [comment]
Good to hear, apparently something changed in the addon working which caused the old URL to be invalid.
